### ISSUE TITLE
Arm64 images not being published to ghcr

### ISSUE THREAD

**Issue by samueltorres** [parca-dev/parca-agent#418]
I was trying to try out parca-agent on a kind cluster in my Mac M1, unfortunately I couldn't pull the Arm64 image because it doesn't seem to be published.

See https://github.com/parca-dev/parca-agent/pkgs/container/parca-agent/22746026?tag=main-8238b7b2 for reference.

```bash
src/github.com/parca-dev
❯ docker pull --platform=linux/arm64 ghcr.io/parca-dev/parca:v0.11.1
v0.11.1: Pulling from parca-dev/parca
Digest: sha256:c7f82b2e1a96b4b7d5607dba8169845f574fa84be2b1882263b7f6e4ebe489ab
Status: Image is up to date for ghcr.io/parca-dev/parca:v0.11.1
ghcr.io/parca-dev/parca:v0.11.1

src/github.com/parca-dev
❯ docker pull --platform=linux/arm64 ghcr.io/parca-dev/parca-agent:v0.8.0
v0.8.0: Pulling from parca-dev/parca-agent
no matching manifest for linux/arm64 in the manifest list entries
```

Should it already be published by the podman integration ? or is anything missing in parca CI ? 

Thanks for this great product 🚀

**Comment by brancz** [comment]
I think https://github.com/parca-dev/parca-agent/pull/415 should fix this.

**Comment by brancz** [comment]
@kakkoyun can answer that best, but I’d be happy to just cut one to see if this is fixed.

**Comment by kakkoyun** [comment]
We have still CI build issues for arm64. I hope to fix them soon. Give us a couple of days, we're still recovering from KubeCon 😅 

cc @v-thakkar

**Comment by brancz** [comment]
I _think_ https://github.com/parca-dev/parca-agent/pull/419 should do the trick

**Comment by kakkoyun** [comment]
@v-thakkar is working on this.

**Comment by kakkoyun** [comment]
This is fixed by #458. With the next major release, the images will be there.
